Makefile

```diff
@@ -70,29 +70,38 @@ unit-test:
 
 .PHONY: build
 build:
-	CGO_ENABLED=0 GOOS=linux go build -a -ldflags ${LDFLAGS} -o _output/secrets-store-csi-driver-provider-azure ./cmd/
+	CGO_ENABLED=0 GOARCH=${ARCH} GOOS=linux go build -a -ldflags ${LDFLAGS} -o _output/${ARCH}/secrets-store-csi-driver-provider-azure ./cmd/
 
 .PHONY: build-windows
 build-windows:
-	CGO_ENABLED=0 GOOS=windows go build -a -ldflags ${LDFLAGS} -o _output/secrets-store-csi-driver-provider-azure.exe ./cmd/
+	CGO_ENABLED=0 GOARCH=${ARCH} GOOS=windows go build -a -ldflags ${LDFLAGS} -o _output/${ARCH}/secrets-store-csi-driver-provider-azure.exe ./cmd/
```

Comment:
Will it not result in error? `windows/arm64` is not supported. `cmd/go: unsupported GOOS/GOARCH pair windows/arm64`

Reply:
The default ARCH is `amd64`

https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/0081a34f24794d19307049fef78b1917f8ba4bdf/Makefile#L43 and we don't use `arm64` for windows

https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/0081a34f24794d19307049fef78b1917f8ba4bdf/Makefile#L36